lib/WebSocket.js

```diff
@@ -262,6 +265,14 @@ WebSocket.prototype.send = function send(data, options, cb) {
 
       if (typeof cb === 'function') cb(error);
     });
+  } else if (Array.isArray(data)) {
+    // send array as iovecs
+    // using fin just like sending a stream but there's no need to
+    // wait for the stream data to be ready, so push all to the sender.
+    for (var i = 0; i < data.length; ++i) {
+      options.fin = (i + 1 === data.length);
+      this._sender.send(data[i], options, options.fin ? cb : undefined);
```

Comment:
Hi @ronkorving Thanks for the comment.

you mean in the case that `typeof(options.binary)==='undefined'`? 
sure, that can be done of course.

but there's a weird thing about this case that on the receiver side the message always appears with `flags.binary===true` even when all iovecs are strings and `options.binary===false` so I'm not sure how much it makes sense to support iovecs send with 'options.binary=false at all? or is that somehow expected that the receiver might be flagged that the message is binary although was sent as non binary?